src/Core/Domain/AbstractBulkCommandHandler.php

```diff
@@ -72,16 +71,16 @@ protected function handleBulkAction(array $ids, string $exceptionToCatch): void
     }
 
     /**
-     * @param Throwable[] $coughtExceptions
+     * @param Throwable[] $caughtExceptions
      *
      * @return BulkCommandExceptionInterface
      */
-    abstract protected function buildBulkException(array $coughtExceptions): BulkCommandExceptionInterface;
+    abstract protected function buildBulkException(array $caughtExceptions): BulkCommandExceptionInterface;
 
     /**
-     * @param mixed $id
+     * @param object $id
      */
-    abstract protected function handleSingleAction($id): void;
+    abstract protected function handleSingleAction(object $id): void;
```

Comment:
```suggestion
    abstract protected function handleSingleAction($id): void;
```
I think since this one is abstract and very generic we should keep the parameter type as generic as possible, this way it could be used with an array of `int` as well, or an array of `string` (uuids)

If you really want to add a type then you should use `mixed` instead just like you did for the `supports` method, and it will keep a better consistency The two methods relying on `$id` should have the same type for this same parameter